<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryParamsForm</class>
 <widget class="QWidget" name="QueryParamsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>555</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnAdd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/bullet_add.png</normaloff>:/icons/bullet_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemove">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/bullet_delete.png</normaloff>:/icons/bullet_delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnClear">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/cross.png</normaloff>:/icons/cross.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvParams">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
  <action name="aRemoveParam">
   <property name="text">
    <string>Remove param</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
